Fill.py

```diff
@@ -196,6 +196,11 @@ def fill_restrictive(multiworld: MultiWorld, base_state: CollectionState, locati
 
     if not allow_partial and len(unplaced_items) > 0 and len(locations) > 0:
         # There are leftover unplaceable items and locations that won't accept them
+        players = set(item.player for item in unplaced_items) | set(loc.player for loc in locations)
+        for player in sorted(players):
+            items = [item for item in unplaced_items if item.player == player]
+            locs = [loc for loc in locations if loc.player == player]
+            multiworld.worlds[player].fill_error(items, locs)
```

Comment:
So this would get called even for players that had no unplaced items or locations, right? Seems a bit weird, but it's not bad or anything